thirdeye/thirdeye-pinot/config/data-sources/cache-config.yml

```diff
@@ -9,12 +7,28 @@ centralizedCacheSettings:
   # if inserting data points individually, max number of threads to spawn to parallel insert at a time
   maxParallelInserts: 10
   # which store to use
-  cacheDataStoreName: couchbase
+  cacheDataStoreName: 'couchbase'
   cacheDataSources:
     couchbase:
-      # host can just be localhost if you're using this locally.
-      host: your_host
-      authUsername: your_username
-      authPassword: your_password
-      bucketName: your_bucket_name
-    # add your store of choice here
\ No newline at end of file
+      className: org.apache.pinot.thirdeye.detection.cache.CouchbaseCacheDAO
+      config:
+        useCertificateBasedAuthentication: falses
+        # at least 1 host needed
+        hosts:
+          - 'host1' # ex. http://localhost:8091
+          - 'host2' # ex. http://localhost:8092
+          - 'host3' # ex. http://localhost:8093
+          - 'host4' # and so on...
+        bucketName: 'your_bucket_name'
+        # if using certificate-based authentication, authUsername and authPassword values don't matter and won't be used
+        authUsername: 'your_bucket_user_username'
+        authPassword: 'your_bucket_user_password'
+        enableDnsSrv: true
+        # certificate based authentication is only available in Couchbase enterprise edition. 
+        keyStoreFilePath: 'key/store/path/keystore_file' # e.g. '/var/identity.p12'
+        # by default, Java uses this password to enable empty passwords for certificates.
+        keyStorePassword: 'work_around_jdk-6879539'
```

Comment:
Is it a true password?

Reply:
No, it is used as a placeholder to designate empty/not used password. See https://bugs.java.com/bugdatabase/view_bug.do?bug_id=6879539

If there is a password associated with the keystore file, user can set it here.

Reply:
Thanks for the explanation.

Reply:
I added some more context in a comment, since it's not very intuitive. Thanks for pointing that out